packages/hoprd/src/api/v3/paths/tickets/statistics.ts

```diff
@@ -51,6 +52,7 @@ GET.apiDoc = {
           schema: {
             type: 'object',
             properties: {
+              pending: { type: 'number', description: 'Not used. Always 0.' },
```

Comment:
Should be removed in the 2.1, where API should be normalized for 3.0.

Reply:
yes